webapp/channels/src/components/profile_popover/profile_popover_controller.tsx

```diff
@@ -76,6 +82,10 @@ interface Props<TriggerComponentType> {
 }
 
 export function ProfilePopoverController<TriggerComponentType = HTMLSpanElement>(props: Props<TriggerComponentType>) {
+    const intl = useIntl();
+
+    const profileAriaLabel = intl.formatMessage({id: 'profile_popover.aria_label', defaultMessage: 'profile popover'});
```

Comment:
@hmhealey do we need to mention popover specifically? i thought the voiceover says its a popover or dialog